packages/design-system/src/components/tooltip/index.js

```diff
@@ -133,6 +134,17 @@ function Tooltip({
 
     setArrowDelta(delta);
   }, []);
+  // When near the bottom of the viewport and the tooltip is placed on the bottom we want to force the tooltip to the top as to not
+  // cutoff the contents of the tooltip.
+  const positionPlacement = useCallback(() => {
+    const tooltipElBoundingBox = tooltipRef.current?.getBoundingClientRect();
+    if (
+      placement.startsWith('bottom') &&
+      tooltipElBoundingBox?.bottom > window.visualViewport.height
+    ) {
+      dynamicPlacement.current = PLACEMENT.TOP;
```

Comment:
In the future we may want to do this for placement 'top' tooltips too. For now I don't think that's necessary though.